arbos/programs/programs.go

```diff
@@ -197,9 +197,10 @@ func (p Programs) CallProgram(
 
 	// pay for program init
 	cached := program.cached || statedb.GetRecentWasms().Insert(codeHash, params.BlockCacheSize)
-	if cached {
+	if cached || params.Version > 1 { // in version 1 cached cost is part of called cost
```

Comment:
```
// in version 1 cached cost is part of called cost
```
Since we are separating `cached_init_cost` from `init_cost` here, should we say `... part of init cost` instead? As the total cost is callCost, I felt this comment to be a bit confusing

Reply:
Yeah I think it should say "init cost" both here and in the comment in `arbitrator/prover/src/binary.rs`

Reply:
Actually one other change here, I think it should be `program.version` not `params.Version`, though this doesn't matter in practice rn as we only support one version. I'll open a new PR.